kit/service/transport.go

```diff
@@ -131,7 +132,7 @@ func (r *Router) MountRoute(route Route) *Router {
 		_, handler = newrelic.WrapHandle(r.newrelicApp, route.GetPath(), handler)
 	}
 
-	r.prefixMux.Methods(route.GetMethod()).Path(route.GetPath()).Handler(handler)
+	r.prefixMux.Methods(route.GetMethod()).Path(route.GetPath()).Handler(handlers.CORS()(handler))
```

Comment:
By default this lib does not allow PUT requests:
```
defaultCorsMethods = []string{"GET", "HEAD", "POST"}
defaultCorsHeaders = []string{"Accept", "Accept-Language", "Content-Language", "Origin"}
```

We also need to allow our custom headers: 'X-Connect-Client-Type' and 'X-Connect-Client-Version'